src/components/Form.js

```diff
@@ -1,48 +0,0 @@
-import React from "react";
-
-const Form =({input, setInput, todos, setTodos, setFilter})=>{
-
-    const inputHandler  = (e) =>{
-       console.log(e.target.value);
-        setInput(e.target.value);
-    }
-    
-    const formSubmit = (e) => {
-        e.preventDefault()
-        setTodos([
-                ...todos, {text: input, completed: false, id: Math.random()*1000 },
-                 ]);
-                 setInput("");
-    };
```

Comment:
I can add empty todo.

Instead using math random which doesn't give you unique ID you can use for instance uuid